feat(aps): add survey banner (#3781)
scaleway/docs-content: tutorials/install-github-actions-runner-mac/index.mdx

  posted: 2024-01-31
---

<ClickableBanner
  productLogo="appleSilicon"
  title="Preconfigured GitHub or GitLab Runners"

>   description="Scaleway is planning to offer pre-installed GitHub and GitLab runners on Mac Mini devices. These runners would be specifically optimized for macOS workflows, providing an efficient and seamless integration into your CI/CD pipelines."

Reviewer: ```suggestion
  description="Scaleway plans to offer pre-installed GitHub and GitLab runners on Mac Mini devices. These runners would be specifically optimized for macOS workflows, providing an efficient and seamless integration into your CI/CD pipelines."
```

  url="https://forms.gle/oWGkXdtDL5WBU5569"

>   label="Fill out the form, if you are interested in learning more" />

Reviewer: ```suggestion
  label="Fill out the form to learn more" />
```
